07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,166 @@
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Drop Down </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                  <a class="dropdown-item" href="#">Action</a>
+                  <a class="dropdown-item" href="#">Another action</a>
+                  <a class="dropdown-item" href="#">Something else here</a>
+                </div>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#contact">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </section>
+
+    <!-- Body of Content -->
+    <section class="main">
+      <div class="container text-center">
+        <h1 class="mt-5 pt-5">Simple Systeme Fordiscount</h1>
+        <h5 class="mt-4">Integer Cursus Bibendum Ague Ac Cursus</h5>
+      </div>
+      <div class="row m-5">
+        <div class="col-lg-4 text-center mb-sm-5">
+          <img src="./assets/img/svg/cloud.svg" alt="" width="70" />
+          <h6 class="font-weight-bold my-4">INTRODUCING WHATSAPP</h6>
+          <div class="mb-4">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi illo veritatis totam dicta laborum, velit voluptate perferendis similique sunt qui quam saepe possimus. Corporis aspernatur nulla eligendi fugit, neque dolorem.
+          </div>
+          <a href="#" class="text-decoration-none" style="color: black">Read More</a>
+        </div>
+        <div class="col-lg-4 text-center mb-sm-5">
+          <img src="./assets/img/svg/planet.svg" alt="" width="70" />
+          <h6 class="font-weight-bold my-4">USER FRIENDLY INTERFACE</h6>
+          <div class="mb-4">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi illo veritatis totam dicta laborum, velit voluptate perferendis similique sunt qui quam saepe possimus. Corporis aspernatur nulla eligendi fugit, neque dolorem.
+          </div>
+          <a href="#" class="text-decoration-none" style="color: black">Read More</a>
+        </div>
+        <div class="col-lg-4 text-center mb-sm-5">
+          <img src="./assets/img/svg/asteroid.svg" alt="" width="70" />
+          <h6 class="font-weight-bold my-4">BUILD THE APP EVERYONE LOVE</h6>
+          <div class="mb-4">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi illo veritatis totam dicta laborum, velit voluptate perferendis similique sunt qui quam saepe possimus. Corporis aspernatur nulla eligendi fugit, neque dolorem.
+          </div>
+          <a href="#" class="text-decoration-none" style="color: black">Read More</a>
+        </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)

Reply:
oke, thanks infonya mas.